src/framework/global/dataformatter.cpp

```diff
@@ -44,48 +44,29 @@ String DataFormatter::formatTimeSince(const Date& dateTime)
         return mtrc("global", "Today");
     }
 
-    if (days == 1) {
-        return mtrc("global", "Yesterday");
```

Comment:
I strongly recommend not changing the implementation of this function

Reply:
Even if we decide to keep "Yesterday" etc. as a separate string, we will still need to use the `%n` way for the number of days. This article has a very good explanation why it's necessary: https://doc.qt.io/archives/qq/qq19-plurals.html

By the way, looking at how complex the rules for some languages are, I think we _do_ need to keep "Yesterday" as a separate string. Some languages use the "singular" form not only for `n == 1`.

Reply:
running out of time, holidays starting and 2 weeks AFK...